docs/Platform-Architecture/permissions-model.md

```diff
@@ -0,0 +1,164 @@
+---
+title: ADP Permissions Model
+summary: Overview of the permissions model in the Azure Development Platform
+uri: https://defra.github.io/adp-documentation/Platform-Architecture/permissions-model/
+authors:
+    - Logan Talbot
+date: 2024-03-14
+weight: -11
+---
+
+# ADP Permissions Model
+
+This page contains an overview of the roles and permissions within ADP (Azure Development Platform). It outlines the different roles such as Platform User, Technical Team Member, Delivery Team Admin, Delivery Programme Admin, and ADP Admin, along with their respective descriptions and responsibilities. Explains the permissions associated with each role in the ADP Portal, Azure DevOps, and GitHub. It describes how permissions are stored in a database and Azure AD using AAD groups. Users are assigned to specific groups based on their roles, granting them the necessary permissions in the ADP Portal, GitHub, Azure, and Azure DevOps.
+
+## ADP Roles
+
+The table below details the roles in the Platform, their scope and description:
+
+
+| Role | Scope | Description |
+|------|-------|-------------|
+| Platform User | Platform | A user of the ADP Platform, who has access to the ADP Portal and can be a member of a Delivery Project or Programme. To do this, they must have a Cloud or DefraGovUK Account. |
+| Technical Team Member | Delivery Project | Tech Lead, Tester, Developer, or Architect on the Delivery Project team. |
+| Delivery Team Member | Delivery Project | Member of the Delivery Project team. |
+| Delivery Team Admin | Delivery Project | Tech lead and/or Delivery Manager for the Delivery Project team. |
+| Delivery Programme Admin | Delivery Programme | Administers Delivery Programmes within the ADP Portal. |
+| ADP Admin | Platform | ADP Platform Engineering delivery team member. |
+| CCoE Engineer | Organization | Cloud Center of Excellence engineer. |
+
+!!! info
+    **Please note:** if a user holds multiple roles, they will be granted the permissions associated with their all roles. This ensures that they have all the access rights and privileges of the most significant role they possess.
```

Comment:
**Please Note:** When a user holds multiple roles, they will receive the combined permissions associated with all their roles. This ensures that they have access to all the rights and privileges granted by the most significant role they possess. Essentially, the role with the highest level of permissions takes precedence.